src/main/scala/rx/lang/scala/Observable.scala

```diff
@@ -5042,6 +5043,24 @@ object Observable {
   }
 
   /**
+   * Converts a `Try` into an `Observable`.
+   *
```

Comment:
Not sure who makes them, but we may or may not want a marble diagram here, just as with most of the other operators/factory methods. If so, I'd be happy to put a `<img .../>` in here!